ReactWindows/ReactNative/Views/WebView/ReactWebViewManager.cs

```diff
@@ -0,0 +1,289 @@
+                    { "goBack", CommandGoBack },
+                    { "goForward", CommandGoForward },
+                    { "reload", CommandReload },
+                };
+            }
+        }
+
+        /// <summary>
+        /// Sets whether a JavaScript is enabled.
+        /// </summary>
+        /// <param name="view">a webview instance.</param>
+        /// <param name="enabled"></param>
+        [ReactProp("javaScriptEnabled")]
+        public void SetJavaScriptEnabled(Windows.UI.Xaml.Controls.WebView view, bool enabled)
+        {
+            view.Settings.IsJavaScriptEnabled = enabled;
+        }
+
+        /// <summary>
+        /// Sets whether the webpage scales to fit the view and the user can change the scale.
+        /// </summary>
+        /// <param name="view">a webview instance.</param>
+        /// <param name="enabled"></param>
+        [ReactProp("scalesPageToFit")]
+        public void SetScalesPageToFit(Windows.UI.Xaml.Controls.WebView view, bool enabled)
+        {    
+            // No implementation       
+        }
+
+        /// <summary>
+        /// Used on Android only, controls whether DOM Storage is enabled or not.
+        /// </summary>
+        /// <param name="view">a webview instance.</param>
+        /// <param name="enabled"></param>
+        [ReactProp("domStorageEnabled")]
+        public void SetDomStorageEnabled(Windows.UI.Xaml.Controls.WebView view, bool enabled)
+        {
+            // No implementation
+        }
+
+        /// <summary>
+        /// Sets the user-agent for this webview.
+        /// </summary>
+        /// <param name="view">a webview instance.</param>
+        /// <param name="userAgent"></param>
+        [ReactProp("userAgent")]
+        public void SetUserAgent(Windows.UI.Xaml.Controls.WebView view, string userAgent)
+        {
+            // No implementation
+        }
+
+        /// <summary>
+        /// Sets the JS to be injected when the webpage loads.
+        /// </summary>
+        /// <param name="view">a webview instance.</param>
+        /// <param name="injectedJavaScript"></param>
+        [ReactProp("injectedJavaScript")]
+        public void SetInjectedJavaScript(Windows.UI.Xaml.Controls.WebView view, string injectedJavaScript)
+        {
+            if (_injectedJS.ContainsKey(view.GetTag()))
+            {
+                _injectedJS.Remove(view.GetTag());
+            }
+            _injectedJS.Add(view.GetTag(), injectedJavaScript);
+        }
+
+        /// <summary>
+        /// Sets webview source.
+        /// </summary>
+        /// <param name="view">a webview instance.</param>
+        /// <param name="source"></param>
+        [ReactProp("source")]
+        public void SetSource(Windows.UI.Xaml.Controls.WebView view, JObject source)
+        {
+            if (source != null)
+            {
+                JToken html, uri;
+
+                if (source.TryGetValue("html", out html) && html.Type == JTokenType.String)
```

Comment:
This is a bit verbose, we can achieve the same think without out parameters with:
var html = source.Value<string>("html");
if (html != null) { ... }

Same goes for below.  I think we can make assumptions about the expected type and except the cast exceptions if for whatever reason the JavaScript contract is invalid.